sympy/combinatorics/perm_groups.py

```diff
@@ -123,10 +123,10 @@ def __new__(cls, *args, **kwargs):
         """The default constructor. Accepts Cycle and Permutation forms.
         Removes duplicates unless ``dups`` keyword is False.
         """
-        args = list(args[0] if is_sequence(args[0]) else args)
         if not args:
             raise ValueError('must supply one or more permutations '
```

Comment:
It could be even possible to avoid ``ValueError``. One can think of the group generated by an empty args list as the symmetric group of the empty set, that consists of the identity element alone. So it might be possible to set
``args = [Permutation()]``
(assuming ``Permutation()`` is acceptable) and then to go on with no error message.